Networking/Networking/Model/ShippingLabel/Packages/Carriers and Rates/ShippingLabelPackageSelected.swift

```diff
@@ -36,6 +45,7 @@ public struct ShippingLabelPackageSelected: Equatable, GeneratedFakeable {
     }
 
     public init(predefinedPackage: ShippingLabelPredefinedPackage, totalWeight: Double, customsForm: ShippingLabelCustomsForm?) {
+        self.id = predefinedPackage.id
```

Comment:
Should we use `default_box` as the ID here to be consistent with the web app too?

Reply:
Same as above, I'll refactor this part in the next PR